forc-pkg/Cargo.toml

```diff
@@ -13,7 +13,7 @@ anyhow = "1"
 fd-lock = "3.0"
 forc-tracing = { version = "0.33.0", path = "../forc-tracing" }
 forc-util = { version = "0.33.0", path = "../forc-util" }
-fuels-types = "0.33"
+fuel-abi-types = "0.1"
```

Comment:
Looks like we just need to update `forc-client` to point to `fuel-abi-types` too, then I think all of the sway repo crates will be on `fuel-abi-types`.

Reply:
Looks like `forc-client` still needs `fuels-types` for `Bech32Address`: https://github.com/FuelLabs/sway/blob/847cbccd32ba0a1a5cc6399e837f43b9b0e4726c/forc-plugins/forc-client/src/ops/tx_util.rs#L14

which is not in `fuel-abi-types` unfortunately.